app/controllers/repositories_controller.rb

```diff
@@ -65,7 +65,7 @@ def create
       create_categories
       create_equipments
       render json: { redirect_uri: "#{repository_path(@user.username, @repository.slug)}" }
-      Repository.reindex
+      # Repository.reindex
```

Comment:
probably worth deleting instead of just commenting. I think this was a solr thing